src/Aspire.Dashboard/Model/DashboardClient.cs

```diff
@@ -119,7 +118,7 @@ GrpcChannel CreateChannel()
                     ClientCertificates = certificates
                 };
 
-                configuration.Bind("ResourceServiceClient:Ssl", httpHandler.SslOptions);
+                configuration.Bind("Dashboard:ResourceServiceClient:Ssl", httpHandler.SslOptions);
```

Comment:
@drewnoakes ?

I think it's used as a simple way to set TLS config in tests without having to explicitly add more config.

Reply:
I think we should own all of the options types and copy settings over explicitly. Less fragile. Can be follow up.

Reply:
What do you think about continuing to do this, but not advertising it in docs? In other words, it's an internal implementaiton detail. A private API.

Then we add new options if people ask for them.

Reply:
Can we preserve the schema?

Reply:
I don't understand. What do you mean?

Reply:
You said don't document it, I assume that to mean, we don't document it because we might change it later so it would be a breaking change if we do.

I'm asking if we know what the existing schema is, and can we preserve it to avoid breaking people.

Reply:
It's the public getters and setters on https://learn.microsoft.com/en-us/dotnet/api/system.net.security.sslclientauthenticationoptions?view=net-8.0

Reply:
I'm worried about this type adding something then suddenly config binding breaks. Or we switch to the source generator then it breaks.